flyway-core/src/main/java/org/flywaydb/core/internal/database/cockroachdb/CockroachDBTable.java

```diff
@@ -70,39 +71,60 @@ public Boolean call() throws SQLException {
     }
 
     protected boolean doExistsOnce() throws SQLException {
-        if (schema.cockroachDB1) {
+        if (schema.cockroachDB1 ) {
             return jdbcTemplate.queryForBoolean("SELECT EXISTS (\n" +
                     "   SELECT 1\n" +
                     "   FROM   information_schema.tables \n" +
                     "   WHERE  table_schema = ?\n" +
                     "   AND    table_name = ?\n" +
                     ")", schema.getName(), name);
+        } else if ( !schema.hasSchemaSupport ) {
+            return jdbcTemplate.queryForBoolean("SELECT EXISTS (\n" +
+                    "   SELECT 1\n" +
+                    "   FROM   information_schema.tables \n" +
+                    "   WHERE  table_catalog = ?\n" +
+                    "   AND    table_schema = 'public'\n" +
+                    "   AND    table_name = ?\n" +
+                    ")", schema.getName(), name);
+        } else {
+            // There is a bug in CockroachDB v20.2.0-beta.* which causes the string equality operator to not work as
```

Comment:
This will be fixed in the official 20.2.0 release. So I guess it's a judgment call if this code needs to work with the beta release or not.

Reply:
To properly test the CockroachDB beta release we have to make sure all tooling around it (like for example Flyway or JOOQ) work as expected, therefore it is important that this fix also works with the (latest) beta release. Furthermore, my company is currently relying on features only available in CockroachDB 20.2 (e.g. custom schemas and partial indexes), so we need this to work with the beta release.